os/Makefile.unix

```diff
@@ -544,6 +527,17 @@ endif
 	$(Q) $(call MAKE_BOARD_SPECIFIC_BIN, $(BIN_EXE), $(BIN_EXT))
 	$(Q) python ./tools/set_bininfo.py $(BIN_EXT)
 	$(Q) $(call MAKE_SAMSUNG_HEADER, $(BIN_EXE), $(BIN_EXT))
+
+ifeq ($(CONFIG_INTELHEX_BINARY),y)
+	$(Q) echo "CP: $(BIN_EXE).hex"
+	$(Q) $(OBJCOPY) $(OBJCOPYARGS) --change-addresses $(KERNEL_START) -I binary -O ihex $(BIN).bin $(BIN).hex
+endif
+ifeq ($(CONFIG_MOTOROLA_SREC),y)
+	$(Q) echo "CP: $(BIN_EXE).srec"
+	$(Q) $(OBJCOPY) $(OBJCOPYARGS) -O srec $(BIN) $(BIN).srec
```

Comment:
Does not SREC need the `--change-addresses` option?

Reply:
it needs as per my opinion. But no concrete proof that I found online. Since hex needs this option and is using the same command It should be needed.